packages/react/src/components/Accordion/AccordionHeader/AccordionHeader.tsx

```diff
@@ -39,19 +38,19 @@ export const AccordionHeader = forwardRef<
       ref={ref}
       size='xsmall'
       level={level}
-      className={cl(classes.header, className)}
+      className={cl('accordion__header', className)}
       {...rest}
     >
       <button
         type='button'
-        className={cl(classes.accordionButton, `fds-focus`)}
+        className={cl('accordion__button', `fds-focus`)}
         onClick={handleClick}
         aria-expanded={context.open}
         aria-controls={context.contentId}
       >
         <ChevronDownIcon
           aria-hidden
-          className={classes.expandIcon}
+          className={cl('accordion__expand-icon')}
```

Comment:
Suggest removing redundant use of `clsx` if its not needed :)
```suggestion
          className='accordion__expand-icon'
```